### PR TITLE
Adding babylonian method in JS,Python

### LINKED ISSUE #1026: Babylonian_method in Js and python.
Can i start working on it ??

### PR DESCRIPTION
**Pull request by khushal87**
Fixes #1026  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Adding babylonian method in Javascript,Python
- 

#### Languages Used:
- Js
- Python

#### Files Added:
-  Babylonian_Method/Babylonian_Method.js
-  Babylonian_Method/Babylonian_Method.py

### CODE REVIEW

File: Babylonian_Method/Babylonian_Method.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    var num=number;
+    var guess=1;
+    var accuracy=0.000001;
+    while(num-guess > accuracy)
+    {
+        num=(num+guess)/2;
+        guess=number/num;
+    }
+    return num;
+}
```
Comment (KanchanThareja) [inline Babylonian_Method/Babylonian_Method.js]: Add more comments to your logic to explain your code.

File: Babylonian_Method/Babylonian_Method.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+//        b) Set guess = number/guess */
+
+//Function for square_root 
+function square_root(number)
+{
+    
+    var num=number;
+    var guess=1;
+    var accuracy=0.000001;
+    while(num-guess > accuracy)
```
Comment (KanchanThareja) [inline Babylonian_Method/Babylonian_Method.js]: Also add some space around " = " operator in both files.

File: Babylonian_Method/Babylonian_Method.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+//    3. Gives the accuracy of decimal places
+//    4. Do following until desired approximation is achieved.
+//        a) Get the next approximation for root using average of num and guess
+//        b) Set guess = number/guess */
+
+//Function for square_root 
+function square_root(number)
+{
+    
+    var num = number;
```
Comment (KanchanThareja) [inline Babylonian_Method/Babylonian_Method.js]: remove line 12 and code looks great

**Comment by khushal87** [comment]
done

File: Babylonian_Method/Babylonian_Method.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#       b) Set guess = number/guess */
+
+#Function for square_root
+def square_root(number):
+    num = number
+    guess = 1
+    accuracy = 0.000001
+    while(num-guess > accuracy):
+        num = (num+guess)/2
+        guess = number/num
```
Comment (SakshiShreya) [inline Babylonian_Method/Babylonian_Method.py]: Add space around operators.

File: Babylonian_Method/Babylonian_Method.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    return num
+
+#To check if function is working
+n = 3;
```
Comment (SakshiShreya) [inline Babylonian_Method/Babylonian_Method.py]: Remove `;`

**Comment by khushal87** [comment]
done @SakshiShreya

**Comment by SakshiShreya** [comment]
Please squash all your commits.
Code looks and works fine.

**Comment by khushal87** [comment]
done @SakshiShreya

**Pull request merged**